pages/linux/nsenter.md

```diff
@@ -4,14 +4,18 @@
 > Particularly useful for docker images or chroot jails.
 > More information: <https://github.com/jpetazzo/nsenter/>.
 
-- Run command in existing processes network namespace:
+- Run a command in the net namespace of {{pid}}:
```

Comment:
what is `net`? an abbreviation of network?

Reply:
Correct; will amend this to be the full form.